tedana/workflows/t2smap.py

```diff
@@ -216,7 +216,7 @@ def t2smap_workflow(data, tes, mask=None, fitmode='all', combmode='t2s',
     # anything that is 10x higher than the 99.5 %ile will be reset to 99.5 %ile
     cap_t2s = stats.scoreatpercentile(t2s_limited.flatten(), 99.5,
                                       interpolation_method='lower')
-    LGR.debug('Setting cap on T2* map at {:.5f}'.format(cap_t2s * 10))
+    LGR.debug('Setting cap on T2* map at {:.5f}ms'.format(cap_t2s * 10))
```

Comment:
Should the logger report milliseconds if the output (and potential input) are in seconds ? Not sure if that will be confusing to an end-user.

Reply:
Good thinking. Changed now.